action/protocol/staking/validations_test.go

```diff
@@ -84,7 +84,8 @@ func TestProtocol_ValidateCreateStake(t *testing.T) {
 		gasLimit  uint64
 		nonce     uint64
 		// expected results
-		errorCause error
+		newError      error
```

Comment:
revert change in this file

this is to test validateCreateStake(), NewCreateStake() should return nil. In other words, we are not testing NewCreateStake() here

Reply:
NewCreateStake() is tested in stakecreate_test.go